src/common/ImageAvatar.js

```diff
@@ -15,14 +15,17 @@ const styles = StyleSheet.create({
 });
 
 export default({ avatarUrl, size, status, isCircular, onPress }) => {
-  const imageStyle = {
+  const touchableStyle = {
```

Comment:
These should both go up in the `styles` object (`StyleSheet.create`). Mind moving them over?